<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HedyLamarrGui</class>
 <widget class="QMainWindow" name="HedyLamarrGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="StabilizationPlot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>1031</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_xCurrentCentroid">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentX">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentY">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>210</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_yCurrentCentroid">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_homeX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_xHomeCentroid">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_homeY">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_yHomeCentroid">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_homeCentroid">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Home centroid</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentCentroid">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current centroid</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_North">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_HedyResponse">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>60</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_West">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_South">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>190</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_East">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB_stepSize">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_stepsize">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Step size </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_initialNSoffset">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_initialEWoffset">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_initialNSoffset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N/S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_initialEWoffset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E/W</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentOffset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Current offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentEWoffset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E/W</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_currentNSoffset">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N/S</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_currentNSoffset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_currentEWoffset">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_ASAreminder">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(from ASA software)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_LogHedyLamarr">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>421</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_arcsec">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>(arcsec)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_setInitialOffset">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Set initial offset </string>
    </property>
   </widget>
   <widget class="QPushButton" name="PB_goToOffset">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Go to offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_goToOffsetNS">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>N/S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_goToOffsetEW">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E/W</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_goToOffsetEW">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LE_goToOffsetNS">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>20</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QFrame" name="stabilizationFrame">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>421</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_deadTime">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Dead time (s)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_integrationTime">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Integration time (s)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_proportionalGain">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Proportional gain</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_stabilizationParameters">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Stabilization parameters</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SB_deadTime">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="SB_integrationTime">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="dSB_proportionalGain">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>60</y>
       <width>51</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_maxMotion">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max. motion (px)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="SB_maxMotion">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>48</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHedyLamarr_GUI">
    <property name="title">
     <string>Hedy Lamarr GUI</string>
    </property>
   </widget>
   <addaction name="menuHedyLamarr_GUI"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
